frontend/src/employee-frontend/components/messages/Sidebar.tsx

```diff
@@ -101,14 +100,16 @@ function Accounts({ accounts, setSelectedReceivers }: AccountsParams) {
     useContext(MessageContext)
 
   const [personalAccount, groupAccounts, unitOptions] = useMemo(() => {
-    const personalAccount = accounts.find(isPersonalMessageAccount)
-    const groupAccounts = accounts.filter(isGroupMessageAccount)
+    const personalAccount = accounts.find((a) => a.account.type === 'PERSONAL')
+    const groupAccounts = accounts.filter((a) => a.account.type === 'GROUP')
```

Comment:
Nit: probably want to do the casting to `NestedGroupMessageAccount` here. Alternatively, you could use type guards here for extra niftyness.